__tests__/app.test.js

```diff
@@ -236,3 +261,151 @@ describe("GET /api/articles/:article_id/comments", () => {
 			});
 	});
 });
+describe("POST /api/articles/:article_id/comments", () => {
+	it("should send the correct status code back (201) when passed the correct comment object ", () => {
```

Comment:
this is handled by the following test - all we need to do is add the status codes we're expecting into the test descriptions e.g. 201: test description here